lib/crypto/eosdart_ecc/src/big_int_encoder.dart

```diff
@@ -0,0 +1,32 @@
+///
```

Comment:
These were the offending functions - pointycastle broke these in version 2.x - by taking the same methods and making them work differently. I guess they didn't think that outside libraries would depend on these!